app/components/common/Feedback.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import { showReportDialog } from '@sentry/browser';
+import { captureException, lastEventId } from '@sentry/react';
+import { v4 as uuidv4 } from 'uuid';
+import { smColors } from '../../vars';
+
+const Container = styled.div`
+  padding: 4px 12px;
+  bottom: 10px;
+  display: flex;
+`;
+
+const Button = styled.div`
+  font-size: 12px;
+  text-decoration: underline;
+  border: 0;
+  color: ${({ theme }) =>
+    theme.isDarkMode ? smColors.lightGray : smColors.darkGray};
+  cursor: pointer;
+`;
+
+const FeedbackButton = () => {
+  return (
+    <Container>
+      <Button
+        onClick={() => {
+          captureException(`client_report_${uuidv4()}`);
+          showReportDialog({ eventId: lastEventId() });
```

Comment:
Also, it takes some time for this function to render a feedback form, so:
1. It frustrating that nothing changed after clicking few times
2. I'd suggest to introduce a state for this component and change the button to spinner on click (there exists `onLoad` callback in `showReportDialog` to switch the state back). So it will also avoid sending `captureException` few times, while User trying to open this dialog :)